Documentation improvements (#145)
Stormbase/django-otp-webauthn: docs/reference/identifying_passkeys.rst

.. _ref_identifying_passkeys:

Reference: Identifying Passkeys using ``django.contrib.identify``

> =================================================================

Reviewer: ```suggestion
Identifying Passkeys using ``django.contrib.identify``
========================================
```

.. versionadded:: v0.11.0

.. sidebar:: Related links

>     :ref:`howto-identifying-passkeys`

Reviewer: ```suggestion
    :ref:`identify-passkeys`
```

This page provides an overview of available functionality in the ``django_otp_webauthn.contrib.identify`` module, which helps you identify the :term:`authenticator` that created a Passkey. This is useful context to give your users when presenting them with a list of their registered Passkeys.

> For detailed instructions on how to install and configure the module, please refer to the :ref:`howto-identifying-passkeys` guide.

Reviewer: ```suggestion
For detailed instructions on how to install and configure the module, see :ref:`identify-passkeys`.
```
